test/public/defaults.js

```diff
@@ -478,43 +487,54 @@ module.exports.waitForTableDataReload = (page, triggerFunction) => Promise.all([
 
 /**
  * Tests whether sorting of main table by column with given id works properly
- * It required there are a least two rows in the table
+ * It is required there are a least two rows in the table
  * @param {puppeteer.Page} page the puppeteer page
  * @param {string} columnId subject column id
  * @return {Promise<void>} promise
  */
-module.exports.testTableAscendingSortingByColumn = async (page, columnId) => {
-    // Expect a sorting preview to appear when hovering over a column header
-    await page.waitForSelector(`th#${columnId}`);
+module.exports.testTableSortingByColumn = async (page, columnId) => {
+    // Expect a sorting preview to appear when hovering over column header
+    await page.waitForSelector(`th#${columnId}`, { timeout: 250 });
     await page.hover(`th#${columnId}`);
     const sortingPreviewIndicator = await page.$(`#${columnId}-sort-preview`);
     expect(Boolean(sortingPreviewIndicator)).to.be.true;
 
-    // Sort by name in an ascending manner
-    const columnHeader = await page.$(`th#${columnId}`);
-    await this.waitForTableDataReload(page, () => columnHeader.evaluate((button) => button.click()));
+    const notOrderData = await this.getAllDataFields(page, columnId);
```

Comment:
We should try to avoid fetching pptr stuff because it's slow.
Would it work to first fetch `targetColumnValues` then something like `notOrderedData = [...targetColumnValues]`